### DISCUSSION TITLE
Custom Fire button

### DISCUSSION THREAD

**Discussion by LASERTAL** [arkypita/LaserGRBL#1452] [Q&A]
Hi , i was wondering if I can customize a fire button that will not have and "Laser exposure timeout" msg that stops the laser after a while.

Im trying to engrave free handed on wood (like a brush) for a project and i use a code i learnd here from stuartB4 (thanks)
that goes like this:
$32=0
M4 S255

![image](https://user-images.githubusercontent.com/83701508/117126743-e2b94800-ada3-11eb-80c2-b09e389aa48e.png)

but when i use higher power like 3000 it turns off after 5 seconds and says "Laser exposure timeout!"
I need it to be limitless, is it possible?

![dddd](https://user-images.githubusercontent.com/83701508/117127626-fc0ec400-ada4-11eb-84ab-70c70c7d037b.jpg)

**Comment by gmmanonymus111** [comment]
Since it is a built in feature of Ortur lasers, you can't turn it off with a LaserGRBL button. Try asking Ortur support if it's possible to turn it off in their firmware.

**Reply by arkypita** [reply]
Correct: It is a safety feature of Ortur firmare, cannot be changed in LaserGRBL

IMHO: It can be annoying, but it is a very important safety feature because a burning laser that does not move can easily cause the material to ignite.
